asa_parser/tests/test_parser.py

```diff
@@ -70,9 +70,11 @@ def test_show_process_cpu_hog(self):
         self.assertIn('"Call Stack":', result)
 
         self.assertIn('"Info":', result)
-        #self.assertIn('"text":', result)
-        #self.assertIn('["Hardware:   FPR-2130"', result)
+        # self.assertIn('"text":', result)
```

Comment:
It looks like you tried to fix a PyCharm style guide warning that there should be a spaces here.  Don't make this change because it's someone else's code and it's likely to cause future merge conflicts.  In a team environment it's better to open a new issue and assign it to the person that owns the code if you'd like to have it fixed.